specs/fault-dispute-game.md

```diff
@@ -128,6 +128,8 @@ The **gindex** of a position $n$ can be calculated as $2^{d(n)} + idx(n)$, where
 - $d(n)$ is a function returning the depth of the position in the Game Tree
 - $idx(n)$ is a function returning the index of the position at its depth (starting from the left).
 
+For example, a position with depth `5` and index `7` would have a gindex of $2^5 + 7 = 128 + 7 = 135$.
+
```

Comment:
Specific examples like this are a bit unusual in specs (though ours aren't always the most formal). That said, the description of generalised indices here isn't the most helpful and it may make sense to show a labelled tree like is done in the SSZ specs: https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md#generalized-merkle-tree-index

Reply:
The spec does have a nice tree a few lines up, but I found it hard to understand this formula without an example. It sounds like the example isn't helpful to others, so I've removed it in 644d28b1c2d0e192a27bdd0b5766e94b157b1abd.

Reply:
Ah cool, I missed that in the diff.